src/lib/utils.ts

```diff
@@ -149,3 +150,12 @@ export const randomizeAvatar = () => {
   const randomIndex = Math.floor(Math.random() * RANDOM_AVATARS.length)
   return RANDOM_AVATARS[randomIndex]
 }
+
+export function generateNanoId(size?: number) {
```

Comment:
how about just assign the default?

```suggestion
export function generateNanoId(size = 7) {
```

Reply:
sure, that would be great, omw